images/manageiq-base/container-assets/container_env

```diff
@@ -9,4 +9,8 @@ function urlescape() {
 safeuser=$(urlescape ${DATABASE_USER})
 safepass=$(urlescape ${DATABASE_PASSWORD})
 
-export DATABASE_URL="postgresql://${safeuser}:${safepass}@${DATABASE_HOSTNAME}:${DATABASE_PORT}/${DATABASE_NAME}?encoding=utf8&pool=5&wait_timeout=5"
+ssl_mode=${DATABASE_SSL_MODE:-prefer}
+database_url="postgresql://${safeuser}:${safepass}@${DATABASE_HOSTNAME}:${DATABASE_PORT}/${DATABASE_NAME}?encoding=utf8&pool=5&wait_timeout=5&sslmode=${ssl_mode}"
+[[ -f /.postgresql/postgresql.crt ]] && database_url="$database_url&sslrootcert=/.postgresql/postgresql.crt"
```

Comment:
To avoid extra "`if`" statements in this file (which I find hard to parse\*), does it make sense try using what was already available for environment variables from postgres?

> The location of the root certificate file and the CRL can be changed by setting the connection parameters `sslrootcert` and `sslcrl` or the environment variables `PGSSLROOTCERT` and `PGSSLCRL`.

(info pull from here:  https://www.postgresql.org/docs/10/libpq-ssl.html#LIBQ-SSL-CERTIFICATES )

I think there is a case to be made that it makes it more clear to do all variables in one fashion, so this isn't a hard stance I am making.  That said, it seems like instead of adding more logic here, it makes more sense to just add another environment variable to the process if `"rootcertificate"` is detected in the `orchestrator.go ` file.

* * *

If we do go this route, we could add a comment here mentioning that root certs are defined via the `orchestrator.go`, and that will set `PGSSLROOTCERT` for you.  Less places the path for the cert is hard coded as well (would only be in the `orchestrator` that way.

Reply:
I'm guessing we don't need to set `PGSSLROOTCERT` since I'm dropping it in the default location.

I figured that if this were on an appliance we would set this in `database.yml` anyway, so I put it in `DATABASE_URL` for consistency.

Reply:
> I figured that if this were on an appliance we would set this in `database.yml` anyway, so I put it in `DATABASE_URL` for consistency.

This rational makes sense, and I am cool with that.  I wasn't hard set on my position here anyway, so I am fine with this staying as it is.